src/server/system_services/schemas/nsfs_config_schema.js

```diff
@@ -35,7 +35,7 @@ const nsfs_node_config_schema = {
         },
         NOOBAA_LOG_LEVEL: {
```

Comment:
should we expose NOOBAA_DEBUG_LEVEL on the schema?

Reply:
yes, On it